.github/workflows/ibis-ci.yml

```diff
@@ -0,0 +1,51 @@
+name: ibis CI
+
+on:
+  pull_request:
+    paths:
+      - 'ibis-server/**'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.number }}
+  cancel-in-progress: true
+
+defaults:
+  run:
+    working-directory: ibis-server
+
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Start Wren JAVA engine
+        working-directory: ./example/duckdb-tpch-example
```

Comment:
This is just a temporary approach, and later we won't need to use the Java engine anymore. We only need its analysis function so that we can disregard other settings.